datameta/static/js/submit.js

```diff
@@ -157,9 +157,13 @@ DataMeta.set_progress_bar = function(uuid, val, classes, text) {
 
 DataMeta.reloadPopovers = function() {
     var popoverTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'))
-    var popoverList = popoverTriggerList.map(function (popoverTriggerEl) {
-        return new bootstrap.Popover(popoverTriggerEl)
-    })
+    try{
+        var popoverList = popoverTriggerList.map(function (popoverTriggerEl) {
+            return new bootstrap.Popover(popoverTriggerEl)
```

Comment:
The error is thrown in this line. If not caught, it will catch the `.catch()` in line 324, resulting the undesired behaviour.

Reply:
> With the fix, I was not able to repeat the problem on ~ 100 test with different files.

Did you try if the popover works? I'd expect that with the try-catch approach the error no longer prevents rendering the table, but that clicking the yellow exclamation mark would now be broken in the same fraction of cases